installer/bundle/bundle_skel_AIX.sh

```diff
@@ -343,16 +439,16 @@ case "$installMode" in
         echo "Updating cross-platform agent ..."
         check_if_pkg_is_installed omi
         if [ $? -eq 0 ]; then
-            pkg_upd $OMI_PKG omi
-            # It is acceptable that this fails due to the new omi being
-            # the same version (or less) than the one currently installed.
-            OMI_EXIT_STATUS=0
+            shouldInstall_omi
+            pkg_upd $OMI_PKG omi $?
```

Comment:
I feel like `shouldInstall_omi` should call ` check_if_pkg_is_installed omi`

Then, we have one function with clear concerns and we don't have to pass the result of shouldInstall_omi to `pkg_upd`